src/context/physical.rs

```diff
@@ -94,106 +104,11 @@ impl SeafowlContext {
             // abstraction (URL: scheme://some-path.to.file.parquet) and it's easier to reuse this
             // mechanism in our case too.
             LogicalPlan::Ddl(DdlStatement::CreateExternalTable(
-                cmd @ CreateExternalTable {
-                    ref name,
-                    ref location,
-                    ..
-                },
+                cmd @ CreateExternalTable { .. },
             )) => {
-                // Replace the table name with the fully qualified one that has our staging schema
```

Comment:
This was refactored out into `prepare_create_external_table` without any logic changes